src/components/PositionedOverlay/tests/PositionedOverlay.test.tsx

```diff
@@ -36,7 +67,10 @@ describe('<PositionedOverlay />', () => {
       const positionedOverlay = mountWithAppProvider(
         <PositionedOverlay {...mockProps} preferredAlignment="left" />,
       );
-      expect(positionedOverlay.prop('preferredAlignment')).toBe('left');
+
+      expect(
```

Comment:
I fixed this test and the test below but we should be relying on visual regression for these